doc/security/security-overview.rst

```diff
@@ -536,76 +536,190 @@ considered and documented.
+  issue to the Assigned state.  Part of triage will be to set the
+  issue's priority.
+
+- Assigned: The issue has been assigned, and is awaiting a fix by the
+  assignee.
+
+- Review: Once there is a Zephyr pull request for the issue, the PR
+  link will be added to a comment in the issue, and the issue moved to
+  the Review state.
+
+- Accepted: Indicates that this issue has been merged into the
+  appropriate branch within Zephyr.
+
+- Release: The PR has been included in a released version of Zephyr.
+
+- Public: The embargo period has ended.  The issue will be made
+  publically visible, the associated CVE updated, and the
+  vulnerabilities page in the docs updated to include the detailed
+  information.
+
+The issues created in this JIRA instance are kept private, due to the
+sensitive nature of security reports.  The issues are only visible to
+certain parties:
+
+- Members of the PSIRT mailing list
+
+- the reporter
+
+- others, as proposed and ratified by the Zephyr Security
+  Subcommittee.  In the general case, this will include:
+
+  - The code owner responsible for the fix.
+
+  - The Zephyr release owners for the relevant releases affected by
+    this vulnerability.
+
+The Zephyr Security Subcommittee shall review the reported
+vulnerabilities during any meeting with more than three people in
+attendance.  During this review, they shall determine if new issues
+need to be embargoed.
+
+The guideline for embargo will be based on: 1. Severity of the issue,
+and 2. Exploitability of the issue.  Issues that the subcommittee
+decides do not need an embargo will be reproduced in the regular
+Zephyr project bug tracking system, and a comment added to the JIRA
+issue pointing to the bug tracking issue.  These issues will be marked
+as being tracked within the Zephyr bug tracking system.
+
+Security sensitive vulnerabilities shall be made public after an
+embargo period of at most 90 days.  The intent is to allow 30 days
+within the Zephyr project to fix the issues, and 60 days for external
+parties building products using Zephyr to be able to apply and
+distribute these fixes.
+
+Fixes to the code shall be made through pull requests PR in the Zephyr
+project github.  Developers shall make an attempt to not reveal the
+sensitive nature of what is being fixed, and shall not refer to CVE
+numbers that have been assigned to the issue.  The developer instead
+should merely describe what has been fixed.
+
+The security subcommittee will maintain information mapping embargoed
+CVEs to these PRs (this information is within the JIRA issues), and
+produce regular reports of the state of security issues.
+
+Each JIRA issue that is considered a security vulnerability shall be
+assigned a CVE number.  As fixes are created, it may be necessary to
+allocate additional CVE numbers, or to retire numbers that were
+assigned.
+
+Each Zephyr release shall contain a report of CVEs that were fixed in
```

Comment:
This paragraph is repeated in the next section.  Suggest to move to para 1 in next section.